src/collective/cover/tiles/carousel.py

```diff
@@ -66,7 +65,10 @@ def populate_with_object(self, obj):
         :type uuids: List of strings
         """
         super(ListTile, self).populate_with_object(obj)  # check permission
-        uuids = ICoverUIDsProvider(obj).getUIDs()
+        if obj.portal_type == 'Collection':
```

Comment:
I guess that this change is because the rule for being addable to a container is no longer limited on contenttype, but only if it can provide a (lead) image and this image can also be providerd by folders,  so folders can no longer be unpacked? 

Just to avoid any misunderstanding My original question was only on the list tile expansion behaviour, for carousels I'm still fine if they're unpacked.  Alas there's not 'one use case to rule them all :-S'